### ISSUE TITLE
the unlabeled samples include the labeled samples

### ISSUE THREAD

**Issue by KK666-AI** [Sleepychord/ImprovedGAN-pytorch#10]
Dear author, thanks for sharing the improved GAN in pytorch. In readme.md, you said: Default configs can train models achieving 98.5% accuracy on test dataset with 100 labeled data(10 per class) and other 59,000 unlabeled data after 100 epochs.
However, according to your implementation, the unlabeled samples are selected as the full train samples of MNIST. It seems they are inconsistent. Which is correct?
